modules/test/playwright/tests/workspaces/liferay-customer-workspace/tests/general/unit/projectUserPermission.spec.ts

```diff
@@ -0,0 +1,190 @@
+	apiHelpersTest,
+	customerApiHelpersTest,
+	customerPagesTest,
+	loginTest()
+);
+
+const accountExternalReferenceCode = 'ERC-001';
+let userEmailAddress: string;
+
+test.afterEach(async ({apiHelpers}) => {
+	const account =
+		await apiHelpers.headlessAdminUser.getAccountByExternalReferenceCode(
+			accountExternalReferenceCode
+		);
+
+	await apiHelpers.headlessAdminUser.deleteUserFromAccountByEmailAddress(
+		account.id,
+		'test@liferay.com'
+	);
+
+	const userAccount =
+		await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+			userEmailAddress
+		);
+
+	await apiHelpers.headlessAdminUser.deleteUserAccount(userAccount.id);
+});
+
+test.beforeEach(async ({apiHelpers, page}) => {
+	await mockOktaApiSession(page);
+	await mockProvisioningApiAssignUser(page);
+
+	const account =
+		await apiHelpers.headlessAdminUser.getAccountByExternalReferenceCode(
+			accountExternalReferenceCode
+		);
+
+	await apiHelpers.headlessAdminUser.assignUserToAccountByEmailAddress(
+		account.id,
+		['test@liferay.com']
+	);
+
+	const rolesResponse = await apiHelpers.headlessAdminUser.getAccountRoles(
+		account.id
+	);
+
+	const accountAdministratorRole = rolesResponse?.items?.filter((role) => {
+		return role.name === 'Account Administrator';
+	});
+
+	await apiHelpers.headlessAdminUser.assignAccountRoles(
+		accountExternalReferenceCode,
+		accountAdministratorRole[0].id,
+		'test@liferay.com'
+	);
+});
+
+test.describe('Project User Permission', () => {
+	test('User can only see things related to User role', async ({
+		apiHelpers,
+		customerApiHelpers,
+		homePage,
+		page,
+		projectAttachmentsPage,
+		projectOverviewPage,
+		projectPaaSPage,
+		projectTeamMembersPage,
+	}) => {
+		await projectOverviewPage.goto(accountExternalReferenceCode);
+
+		const accountFlag = await customerApiHelpers.getAccountFlag(
+			accountExternalReferenceCode
+		);
+
+		if (accountFlag === undefined) {
+			await expect(
+				page.getByRole('button', {name: 'Start Project Setup'})
+			).toBeVisible();
+		}
+
+		await projectTeamMembersPage.goto(accountExternalReferenceCode);
+
+		await projectTeamMembersPage.inviteButton.click();
+
+		await projectTeamMembersPage.firstNameField.fill('testfirst');
+
+		await projectTeamMembersPage.lastNameField.fill('testlast');
+
+		userEmailAddress = getRandomString() + '@liferay.com';
+
+		await projectTeamMembersPage.emailField.fill(userEmailAddress);
+
+		await projectTeamMembersPage.roleSelect.click({force: true});
+
+		await projectTeamMembersPage.userRoleOption.click();
+
+		await projectTeamMembersPage.applyButton.click();
+
+		await projectTeamMembersPage.sendInvitationsButton.click();
+
+		await projectTeamMembersPage.goto(accountExternalReferenceCode);
+
+		await expect(page.getByText(userEmailAddress)).toBeVisible();
+
+		const userAccount =
+			await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+				userEmailAddress
+			);
+
+		await apiHelpers.headlessAdminUser.patchUserAccount(userAccount, {
```

Comment:
is this changing the user password from test to test? if it's a new user, should currentPassword just be blank?

Reply:
It's a new user, probable will work just passing the password, but will test to make sure.

Reply:
Worked fine just with password attribute.